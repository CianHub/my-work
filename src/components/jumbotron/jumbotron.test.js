// jumbotron.jumbotron.test.js
import React from "react";
import renderer from "react-test-renderer";
import Jumbosection from "./jumbotron";

test("Jumbotron is rendered", () => {
  const component = renderer.create(<Jumbosection />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
