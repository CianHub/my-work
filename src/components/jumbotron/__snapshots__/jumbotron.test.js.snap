// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron is rendered 1`] = `
<section
  className="text-center"
  id="jumbotron"
>
  <div
    className="container"
  >
    <div
      className="jumbo-animation"
    >
      <canvas
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
  <div
    className="jumbotron"
    id="jumbo"
  >
    <div
      className="overlay container"
    >
      <div
        className="jumbo-heading"
      >
        <h1
          className=""
          id="jumbo-text "
        >
          Hi, I'm
          <strong
            className="jumbo-name"
          >
             Cian O'Gralaigh
          </strong>
           
          <br />
          and I'm a full-stack web developer.
        </h1>
      </div>
      <div
        className="jumbo-btn-div"
      >
        <a
          href="#projects"
        >
          <button
            className="btn "
            id="jumbo-button"
            type="button"
          >
            See My Work 
              
          </button>
        </a>
      </div>
    </div>
  </div>
</section>
`;
